packages/vue-components/src/Dropdown.vue

```diff
@@ -11,7 +11,7 @@
         :class="{'disabled': disabledBool}"
         data-bs-toggle="dropdown"
       >
-        <slot name="header"></slot>
+        <span v-html="tabgroupHeader"></span>
```

Comment:
Hmm I'm not keen on this naming since this is only true for tab groups but not for other headers. 
Could we keep the header name?

Reply:
Sure! The rationale was give greater differentiation in the way the header is passed as a prop instead of relying on slot conversion, but it is evident from the code directly. 

I would be down to try writing some test cases for the tabs! Test cases would also prevent any further regressions until a better way to pass down the header is found.

Reply:
I just realised that setting it as the same name header causes two of the current tests to fail under `dropdown` as the header no longer shows up as an attribute in the html rendered. This is probably because of the slot name and the attribute prop name having the same name.

I could edit the testcases to remove the header, or simply revert to using a separate name again to prevent this issue